modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Select/Select.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```

Comment:
Hey @cgmonte, some files don't automatically add the current year, could you add 2024 here? Thanks!